treeherder/webapp/api/performance_data.py

```diff
@@ -840,6 +850,18 @@ def list(self, request):
                     'new_stddev_pct': new_stddev_pct,
                     'base_retriggerable_job_ids': base_grouped_job_ids.get(base_sig_id, []),
                     'new_retriggerable_job_ids': new_grouped_job_ids.get(new_sig_id, []),
+                    'graphs_link': PerformanceSummary.createGraphsLinks(
+                        {
+                            'originalProject': base_repository_name,
+                            'newProject': new_repository_name,
+                            'originalRevision': base_revision,
+                            'newRevision': new_revision,
+                        },
+                        [{'title': 'subtests', 'to': './comparesubtests%s' % urlencode(params)}],
+                        framework,
+                        PerformanceSummary._get_ph_time_ranges(),
+                        new_sig_id if base_perf_data_values.length == 0 else base_sig_id,
```

Comment:
@beatrice-acasandrei I'm not sure if this is correct, but based on what I saw in compareview.jsx it might be